app/client/package.json

```diff
@@ -9,7 +9,12 @@
         "build": "next build",
         "relay": "relay-compiler --watch",
         "typecheck": "tsc -p tsconfig.prod.json --noEmit",
-        "lint": "eslint \"src/**\""
+        "lint": "eslint \"src/**\"",
+        "test:e2e": "playwright test && yarn allure:generate-report",
+        "test:e2e:ci": "dotenv -v CI=1 -- playwright test && yarn allure:generate-report",
```

Comment:
This line could be `"test:e2e:ci": "dotenv -v CI=1 -- yarn run test:e2e"` I believe.

Reply:
Yeah that might be a bit cleaner